### PR TITLE
VUE-318 lightbox improvements

### PR DESCRIPTION
**Pull request by ryan-ludwig**
- Use prop to set title for consistency. 
- Add aria-labelledby and fix focus-lock for a11y. 
- Remove fixed width from lightboxes to allow narrower content without overrides. 
- Add a css custom property for overriding the title color.
- Remove unused "showCloseButton" property
- Scope lightbox styles

### CODE REVIEW

File: src/pages/Autolending/AutolendingWho.vue [hunk 1]
```diff
@@ -17,30 +17,22 @@
```
Comment (ryan-ludwig) [inline src/pages/Autolending/AutolendingWho.vue:17]: @eddieferrer Would appreciate if you could double check the autolending settings lightboxes to make sure I didn't mess anything up here.

File: src/pages/Autolending/AutolendingWho.vue [hunk 2]
```diff
@@ -17,30 +17,22 @@
...
-									@click.prevent="backToAllOptions"
-								>
-									<kv-icon class="arrow back-arrow" name="small-chevron" :from-sprite="true" />
-									Back to all options</a>
-							</transition>
-							<hr>
-						</div>
-					</template>
 					<div class="who-inputs-wrapper">
+						<transition :name="slideTransition" mode="out-in">
```
Comment (eddieferrer) [inline src/pages/Autolending/AutolendingWho.vue:25]: This is not the same as the lines removed in the slot. When the transition happens, the title should be replaced with the "back to all options" with a slide transition

File: src/pages/Autolending/AutolendingStatus.vue [hunk 3]
```diff
@@ -37,17 +37,10 @@
...
 				<kv-lightbox
 					class="autolending-status-lightbox"
-					:no-padding-bottom="true"
-					:no-padding-top="true"
 					:visible="showLightbox"
+					title="Change your auto-lending status"
 					@lightbox-closed="showLightbox = false"
 				>
-					<template slot="title">
-						<h3>
```
Comment (eddieferrer) [inline src/pages/Autolending/AutolendingStatus.vue:46]: The autolend h3 titles have different styling than the styling of the lightbox titles provided in this PR. Take a look at the dev env for reference.

File: src/pages/Autolending/AutolendingWho.vue [hunk 4]
```diff
@@ -123,7 +115,6 @@
 						</transition>
 					</div>
 					<template slot="controls">
-						<hr>
```
Comment (eddieferrer) [inline src/pages/Autolending/AutolendingWho.vue:126]: The autolend settings lightboxes should have an hr line separating the title and the button area...

File: src/pages/Autolending/SaveButton.vue [hunk 5]
```diff
@@ -14,27 +14,23 @@
 		</kv-button>
 		<kv-lightbox
 			:visible="warningVisible"
-			:no-padding-top="true"
```
Comment (eddieferrer) [inline src/pages/Autolending/SaveButton.vue:17]: These no-padding-* props are being used, not sure they should be removed.

**Comment by ryan-ludwig** [comment]
@eddieferrer There are visual changes to the lightboxes in autolending settings but I feel that they still convey the same hierarchy and are now consistent with the lightboxes throughout the site.

Before:
image
image
image

After:
image
image
image

The final image with the "choose categories" behaves a little differently by not swapping out the title, but in the same vein, I feel the user experience is equivalent and we gain consistency through the site.

**Comment by eddieferrer** [comment]
> @eddieferrer There are visual changes to the lightboxes in autolending settings but I feel that they still convey the same hierarchy and are now consistent with the lightboxes throughout the site.
> 
> Before:
> image
> image
> image
> 
> After:
> image
> image
> image
> 
> The final image with the "choose categories" behaves a little differently by not swapping out the title, but in the same vein, I feel the user experience is equivalent and we gain consistency through the site.

I understand, it wasn't clear to me that the visual changes were intentional in this PR.

I would suggest then that we run this change by design.

**Comment by ryan-ludwig** [comment]
> 
> I understand, it wasn't clear to me that the visual changes were intentional in this PR.
> 
> I would suggest then that we run this change by design.

👍  will do

**Comment by ryan-ludwig** [comment]
Got the go-ahead from design to make these visual changes to autolending settings dialogs. Thanks @eddieferrer for reaching out as well.

**Pull request merged**